boa_engine/src/vm/code_block.rs

```diff
@@ -87,6 +87,32 @@ unsafe impl Trace for CodeBlockFlags {
     empty_trace!();
 }
 
+/// This represents a range in the code that handles exception throws.
+///
+/// When a throw happens, we search for handler in the [`CodeBlock`] using
+/// the [`CodeBlock::find_handler()`] method.
+#[derive(Debug, Clone, Copy)]
+pub(crate) struct Handler {
+    pub(crate) start: u32,
+    pub(crate) end: u32,
+
+    #[allow(unused)]
+    pub(crate) fp: u32,
+    pub(crate) env_fp: u32,
```

Comment:
```suggestion
    pub(crate) sp: u32,
    pub(crate) env_sp: u32,
```

Naming nitpick, there seems to be a lot of frame pointers (`fp`) here, and maybe it would be better to refer to these as an `sp` for `stack pointer` rather than a frame pointer?

Reply:
Yes, probably more correct! the stored `fp` is relative to callframe `fp` that's why I initially named it `fp` (`actual_fp = fp + relative_fp`).

Or maybe just name it to `stack_count` since it's not really a pointer, it's a count after `fp`.

Reply:
Yeah, it's a little hard. `stack_count` would probably also work